source/photo_audio_video_file.md

```diff
@@ -0,0 +1,148 @@
+question types as shown in the following example:
+
+| type  | name        | label                                                       | hint            |
+| :---- | :---------- | :---------------------------------------------------------- | :-------------- |
+| image | house_photo | Take a photo of the housing unit                            |                 |
+| audio | impact      | What has been the impact of the project on your household?  | Record as audio |
+| video | preparation | Record video of the respondent as they prepare the VitaMeal |                 |
+| file  | CV          | Attach your CV                                              |                 |
+
+## Appearance of Photo, audio, video and file question types in web forms and KoboCollect
+
+### Default appearance
+
+![Default appearances](images/photo_audio_video_file/default_appearances.png)
+
+### Advanced appearances for "photo" question type
+
+When adding the “Photo” question type, you can choose from a number of
+appearances (under the question settings). Appearances change the way the
+question is displayed on web forms or on KoboCollect.
+
+![Advanced appearances for photo question type](images/photo_audio_video_file/advanced_appearances_photo.png)
+
+![Advanced appearances](images/photo_audio_video_file/advanced_appearances.png)
+
+### Adding advanced appearances in xlsform
+
+You can specify advanced appearances of the "photo" question in XLSForm under
+the appearance column as shown in the following example:
+
+| type  | name       | label                      | appearance |
+| :---- | :--------- | :------------------------- | :--------- |
+| image | sign       | Sign here                  | signature  |
+| image | drawing    | Draw here                  | draw       |
+| image | annotation | Take an image and annotate | annotate   |
+
+## Background audio recording
+
+KoboToolbox now allows you to record audio in the background when you open the
+form. This can be useful in scenarios for example when you are collecting data
+through focus group discussions and you need an audio of the whole interview for
+analysis later.
+
+You can turn on background audio recording in the Form Builder by clicking on
+Layout & Settings.
+
+![Backgrouns audio](images/photo_audio_video_file/background_audio.png)
+
+In XLSForm, you can add the background recording feature by using the
+background-audio meta question type as follows:
+
+| type             | name             | label |
+| :--------------- | :--------------- | :---- |
+| background-audio | background_audio |       |
+
+<section class="note">
+
+Note: It is not possible to record audio using “Audio” question type while
+background audio recording is underway on the form. When background audio
+recording is on, all “Audio” question types are deactivated.
+
+</section>
+
+Read more about background audio recording
+[on the linked article](recording-interviews.md).
```

Comment:
Rather don't say things like "on the linked article".

> You can read more about background audio recording [here](recording-interviews.md).

Reply:
Done